octrc-scripts/syft.vulnerabilityCsv.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: Tim Emmert <tim.emmert@tngtech.com>
+#
+# SPDX-License-Identifier: MIT
+
+set -euo pipefail
+
+json="$1"
+output="$2"
+
+jq -r '["CVE", "namespace", "dataSource", "severity", "package names", "package versions", "purl"], (.matches[] | {vulnerability, package: [.matchDetails[].searchedBy.package], artifact} | [ .vulnerability.id, .vulnerability.namespace, .vulnerability.dataSource, .vulnerability.severity, ([.package[].name] | join("|")), ([.package[].version] | join("|")), .artifact.purl]) | @csv' "$json"  > "$output"
```

Comment:
you could improve readability a lot with:
```suggestion
jq -r '["CVE", "namespace", "dataSource", "severity", "package names", "package versions", "purl"],
       (.matches[] | {vulnerability, package: [.matchDetails[].searchedBy.package], artifact}
                   | [ .vulnerability.id
                     , .vulnerability.namespace
                     , .vulnerability.dataSource
                     , .vulnerability.severity
                     , ([.package[].name] | join("|"))
                     , ([.package[].version] | join("|")), .artifact.purl]) | @csv' \
    "$json"  > "$output"
```

Reply:
Thanks, I will do that